data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/rewards/GetBlockRewardsTest.java

```diff
@@ -35,14 +35,11 @@
 import tech.pegasys.teku.beaconrestapi.AbstractMigratedBeaconHandlerTest;
 import tech.pegasys.teku.infrastructure.async.SafeFuture;
 import tech.pegasys.teku.infrastructure.http.HttpStatusCodes;
-import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 import tech.pegasys.teku.spec.SpecMilestone;
 import tech.pegasys.teku.spec.datastructures.metadata.ObjectAndMetaData;
 
 public class GetBlockRewardsTest extends AbstractMigratedBeaconHandlerTest {
-  private final BlockRewardData data =
-      new BlockRewardData(
-          123, 123L, 123L, UInt64.valueOf(123), UInt64.valueOf(123), UInt64.valueOf(123));
+  private final BlockRewardData data = new BlockRewardData(123, 123L, 123L, 123L, 123L, 123L);
```

Comment:
better to use unique values so you can identify where they're going... can just be `121,122,123...`